### PR TITLE
[dash] Make Swift code look more idiomatic

### PR DESCRIPTION
**Pull request by jamesderlin**
* Remove semicolons.
* `if (condition)` => `if condition`.

Also fix a reference to a function name.

### CODE REVIEW

**Comment by mit-mit** [comment]
This website page is based on the sample code in https://github.com/flutter/flutter/tree/master/examples/platform_channel_swift, so we probably want to update that first

**Comment by jamesderlin** [comment]
> This website page is based on the sample code in https://github.com/flutter/flutter/tree/master/examples/platform_channel_swift, so we probably want to update that first

Okay, I've made pull request https://github.com/flutter/flutter/pull/21712 to update the sample code.

**Comment by Sfshaza** [comment]
I'm waiting for https://github.com/flutter/flutter/pull/21712 to be resolved before updating this.

**Comment by mit-mit** [comment]
Looks like flutter/flutter#21712 landed 👍. @jamesderlin is this PR up-to-date with the final code that landed there?

**Comment by jamesderlin** [comment]
> Looks like [flutter/flutter#21712](https://github.com/flutter/flutter/pull/21712) landed . @jamesderlin is this PR up-to-date with the final code that landed there?

Yes, and with flutter/flutter#22318.

**Comment by mit-mit** [comment]
@Sfshaza can we go ahead and merge?

File: src/development/platform-integration/platform-channels.md [hunk 1]
```diff
@@ -542,15 +543,15 @@ tied to the channel name `samples.flutter.io/battery`:
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
 
-    let controller : FlutterViewController = window?.rootViewController as! FlutterViewController;
-    let batteryChannel = FlutterMethodChannel.init(name: "samples.flutter.io/battery",
-                                                   binaryMessenger: controller);
+    let controller : FlutterViewController = window?.rootViewController as! FlutterViewController
+    let batteryChannel = FlutterMethodChannel(name: "samples.flutter.io/battery",
```
Comment (mit-mit) [inline src/development/platform-integration/platform-channels.md]: Isn't swift typically formatted with the colons (`:`) all aligned?

Comment (jamesderlin) [reply src/development/platform-integration/platform-channels.md]: That's not uncommon in Objective-C, but I don't see that typically done in Swift.  The [Google Swift style guidelines](https://google.github.io/swift/#horizontal-alignment) say not to do horizontal alignment.  That section doesn't specifically talk about function declarations or function calls, but there are some code examples (Trailing Closures, Delegate Methods) that show no effort to align colons.

**Comment by Sfshaza** [comment]
Sorry, @mit-mit, I missed that it was on dash!  Merging.

**Pull request merged**